components/a11y/blender/recipe.js

```diff
@@ -0,0 +1,65 @@
+import { GEL } from '@westpac/core';
+import React from 'react';
+
+import { VisuallyHidden, SkipLink } from '@westpac/a11y';
+
+export function AllStyles({ brand }) {
+	return (
+		<GEL brand={brand}>
+			<VisuallyHidden>This is screen reader only text</VisuallyHidden>
+
+			<SkipLink href="#content">This is screen reader only text (visible when focussed)</SkipLink>
```

Comment:
Let's make it explicit that this is for styles only:
```suggestion
			<VisuallyHidden>TEXT</VisuallyHidden>
			<SkipLink href="#TEXT">TEXT</SkipLink>
```